Revit_Core_Engine/Convert/Environment/FromRevit/CableTray.cs

```diff
@@ -94,6 +153,22 @@ public static List<BH.oM.MEP.System.CableTray> CableTrayFromRevit(this Autodesk.
             return bhomCableTrays;
         }
 
+        /***************************************************/
+        /****              Private Methods              ****/
+        /***************************************************/
+        
+        [Description("Re-orders converted BHoM Line list to match the direction from the reference MEPCurve.")]
+        private static List<BH.oM.Geometry.Line> MatchRevitOrder(List<BH.oM.Geometry.Line> linesToMatch, MEPCurve reference)
+        {
+            LocationCurve locationCurve = reference.Location as LocationCurve;
+            Curve curve = locationCurve.Curve;
+            BH.oM.Geometry.Point referenceStart = BH.Revit.Engine.Core.Convert.PointFromRevit(curve.GetEndPoint(0));
```

Comment:
You can make use of the extension methods to make the code more readable:
```suggestion
            BH.oM.Geometry.Point referenceStart = curve.GetEndPoint(0).PointFromRevit();
```